### PR TITLE
Create Abbas-Hussain-Muzammil.json

### PR DESCRIPTION
**Pull request by Abbas-Hussain-Muzammil**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2440"></a>

### CODE REVIEW

File: data/Abbas-Hussain-Muzammil.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "name": "Abbas Hussain Muzammil",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Enthusiast | Blogger | Software Engineer",
+  "avatar": "https://github.com/Abbas-Hussain-Muzammil.png",
+// tags
```
Comment (eddiejaoude) [inline data/Abbas-Hussain-Muzammil.json:7]: Comments are not valid json, this is causing the GitHub Action to fail

Comment (Abbas-Hussain-Muzammil) [reply data/Abbas-Hussain-Muzammil.json]: Ok, removed them

**Review by Abbas-Hussain-Muzammil — COMMENTED** [review]
Resolved the errors

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Pull request merged**